pojo-generator-core/src/main/java/uk/gov/justice/generation/SchemaPojoGenerator.java

```diff
@@ -106,7 +106,7 @@ private PluginProvider createPluginProvider(final GeneratorConfig generatorConfi
 
         final ClassNameFactory classNameFactory = new ClassNameFactory(typeNameProvider, typeNamePluginProcessor);
         final JavaGeneratorFactory javaGeneratorFactory = new JavaGeneratorFactory(classNameFactory);
-        final PluginContext pluginContext = new PluginContext(javaGeneratorFactory, classNameFactory, generationContext.getSourceFilename());
+        final PluginContext pluginContext = new PluginContext(javaGeneratorFactory, classNameFactory, generationContext.getSourceFilename(), null);
```

Comment:
The classModifyingPlugins argument is set to null, should they come from the PluginProvider?